src/ngraph/op/add.cpp

```diff
@@ -49,7 +49,7 @@ void op::Add::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVector&
     adjoints.add_delta(y, delta);
 }
 
```

Comment:
I ran into problems the last time I tried this change; we might drop the operator+ part if they show up in merge CI.